engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -90,6 +91,7 @@ public void initialise() {
         new WireframeTrigger(renderingDebugConfig, this);
 
         addDesiredStateChange(new BindFBO(READONLY_GBUFFER, displayResolutionDependentFBOs));
+        addDesiredStateChange(new SetRenderBufferMask(READONLY_GBUFFER, displayResolutionDependentFBOs, true, false, false));
```

Comment:
The reason I chose this name was, the function called internally is fbo.setRenderBufferMask. I can change the name regardless, if you want.

Also, I kept it this way because it's data (fbo), and the settings for the data (true, false, false). But I'll change it :D